system/classes/Kohana/Core.php

```diff
@@ -983,7 +980,7 @@ public static function shutdown_handler()
 			ob_get_level() AND ob_clean();
 
 			// Fake an exception for nice debugging
-			Kohana_Exception::handler(new ErrorException($error['message'], $error['type'], 0, $error['file'], $error['line']));
+			Kohana_Exception::handler(new ErrorException($error['message'], $error['type'], $error['type'], $error['file'], $error['line']));
```

Comment:
Why should the $code and $severity parameters be the same?
http://php.net/manual/en/errorexception.construct.php